### ISSUE TITLE
Using the VF vocab in plain text

### ISSUE THREAD

**Issue by bhaugen** [valueflows/valueflows#160]
These are not exactly the VF vocab, but examples of using predecessor vocabularies in either plain or structured text.

![image](https://cloud.githubusercontent.com/assets/117439/19016922/f3af4294-87ed-11e6-813b-1cc963cd1998.png)

![image](https://cloud.githubusercontent.com/assets/117439/19016925/0a49964e-87ee-11e6-95e8-6a0793d90642.png)

**Comment by bhaugen** [comment]
More examples of text vocabs:
- hashtags
- https://opp.io/
  - taking hashtags several steps beyond, into real sentences
  - syntax: @who should !do #what
  - integrates with gmail, trello, etc
    ![image](https://cloud.githubusercontent.com/assets/117439/19020619/cd0df0c8-8873-11e6-8c4b-f785cc377fba.png)
- reddit [#changetip](https://www.reddit.com/r/changetip/wiki/tipping-on-reddit)
- [#punkmoney](https://github.com/webisteme/punkmoney) (punkmoney.org errors out...)

**Comment by bhaugen** [comment]
As in punkmoney, changetip, and opp.io, using the VF vocab in plain text will require some software that understands the vocab and can find vf:statements in some semi-constrained medium.

For punkmoney, the medium (was) twitter, until twitter knocked them out (I think that's what happened). For changetip, that's reddit. For opp.io, that is any of the channels they integrate with, including their own app.

@elf-pavlik @melvincarvalho will it be possible to base a vf:statement finder on Solid?

**Comment by gcassel** [comment]
Creating consistent standards for the networking of descriptive statements/observations/ event-reports is very important.  I currently have a few main personal concerns on this front, which may be of some relevance to VF's development:
- I advocate that statements about events, if auto-generated, are generated very _simply_ and _predictably_ according to the _direct inputs_ of agents.   For instance:   Elmo would be quite likely to agree with an auto-generated statement that he sold one chocolate chip cookie to Cookie Monster, if Elmo directly accepted payment and delivered a cookie to Mr. Monster.   (Or, if Elmo issued a receipt to Cookie Monster, or signed any sort of written sales agreement.)
- If agents compose their own 'event-report statements' for submission to a network, those statements should be subject to confirmation or denial by peers.   Some of my main personal research and design is related:  p2p ratings/evaluation systems for _reports of events_, as well as the folksonomic _application of descriptive terms and descriptive statements_ to media items of all types. 
- Agent's statements in natural language should only IMO be used in formal machine-read contexts as declarative statements or reports/ testimonies _when agents intend for that to happen_.   (For instance, when agents directly submit texts to such contexts.)   Outside of intentionally formal interactions, people tend to write imprecisely, in ways which they don't desire to be held legally or 'mechanically' accountable for.

**Comment by bhaugen** [comment]
> If agents compose their own 'event-report statements' for submission to a network, those statements should be subject to confirmation or denial by peers. 

If you look at the Conversation for Action protocol
![image](https://cloud.githubusercontent.com/assets/117439/19023753/8f43e80e-88bb-11e6-91d4-323b67ba9520.png)
I think it will be clear that each statement (each speech act) requires a confirming response.

**Comment by gcassel** [comment]
> If you look at the Conversation for Action protocol
> I think it will be clear that each statement (each speech act) requires a confirming response.

Certainly, although that's "only" with respect to direct conversations.  FWIW, in my media networking design and facilitation goals, I seek to integrate the entire scope of what has traditionally (but terribly imperfectly) been done with quantitative evaluation and ratings tools, for p2p intentional online communities which have explicit standards (both "dos and don'ts") for each participant-type.   

Of course I realize that that goes well beyond the intended scope of Value Flows.   It seems like it'd  be wise for VF to focus clearly upon modeling interactions and events which are subject to mutual direct verification.

**Comment by bhaugen** [comment]
> Certainly, although that's "only" with respect to direct conversations.

Work that remains to be done: formalize (if possible) agreement protocols among more than two participants.

**Comment by gcassel** [comment]
> Work that remains to be done: formalize (if possible) agreement protocols among more than two participants.

Within the context of VF's economic modeling and accounting goals, I believe that any agreement protocol should include the **direct written consent** by digital signature (with an agent ID which is _unique, per network_)  of every agreeing agent.   Any contrasting views on that?

Some details certainly need to be added.  For instance, how should one distinguish a generic agreement (a mutual understanding) from commitments and contracts, and how can agents terminate their membership in each type of agreement?   Of course, I've gone on at some length on such matters in [Agreement-Based Organization](https://drive.google.com/open?id=1c_xWEIay-2jyJ3Rqb6OgTxoZBJfjNW4d6w6ukXyeJk4), but not in a way that's properly detailed _or_ distilled for VF.  

Unfortunately, having "conversations for action" related to Agreement-Based Organization has been far from my focus, because of the extended personal gestation (pre-Request For Comments) of my [P2P Digital Media Networking Protocol](https://docs.google.com/document/d/1CKXns9XKySsFGcPGnmKiM5wAWJmbeBWZY6iIljAz6a4/edit?usp=sharing) document.   However, I strongly desire to help VF authors to find consensus on definitions and protocols for agreements.     @bhaugen , I guess you'd like a new Issue for that, whenever I feel ready to help facilitate such a discussion?  (Or have I missed or neglected an existing Issue for that?)

**Comment by bhaugen** [comment]
>  I guess you'd like a new Issue for that, whenever I feel ready to help facilitate such a discussion?

No rush, but yes.

> (Or have I missed or neglected an existing Issue for that?)

Not that I know of.
